src/main/java/microsoft/exchange/webservices/data/core/request/ServiceRequestBase.java

```diff
@@ -440,8 +469,8 @@ protected T readResponse(HttpWebRequest response) throws Exception {
         serviceResponseStream.close();
         memoryStream.flush();
       } else {
-        InputStream responseStream = ServiceRequestBase
-            .getResponseStream(response);
+        InputStream responseStream = readResponseStream(response);
+
```

Comment:
Put a comment in the exception hadnling below that says if this changes, to make sure that the the chagnes are incorporated into readResponseStream func